mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/ExecuteStageRequestUtils.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2022 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.mantisrx.server.master;
+
+import io.mantisrx.server.core.ExecuteStageRequest;
+import io.mantisrx.server.master.config.MasterConfiguration;
+import io.mantisrx.server.master.resourcecluster.TaskExecutorRegistration;
+import io.mantisrx.server.master.scheduler.ScheduleRequest;
+import io.mantisrx.shaded.com.google.common.base.Optional;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+public class ExecuteStageRequestUtils {
```

Comment:
I think we should rename this class to something that doesn't sound like a `*Utils` with a bunch of static methods only.